ui/src/Panels/Producer/Producer.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Starter app Producer
+
+Scenario: As a user, I can start and stop production as I choose
+    Given I have a Producer panel
+    When I start the producer
+    Then the producer is running
+    When I stop the producer
+    Then the producer is not running
+
+Scenario: As a user, I can provide the message value for produced messages
+    Given I have a Producer panel
+    And I set the message value to "A new value"
+    When I start the producer
+    Then the producer has been started with "A new value" as the message value
+    And the message value input is disabled
+
+Scenario: As a user, I can see statistics of what has been produced so far
+    Given I have a Producer panel
+    When I start the producer
+    And responses are returned
+    Then it should display statistics about what has been produced
+
+Scenario: As a user, I am shown error states if production fails
+    Given I have a Producer panel
+    When I start the producer
+    And Error responses are returned
+    Then I should be shown error messages for those production failures
+
+Scenario: As a user, I can see what has been produced so far
```

Comment:
And so I dont forget, a test will need to be added (ala the consumer) to cover the user selecting a message

Reply:
Added